### DISCUSSION TITLE
US house numbers missing (nominatim website vs own installation)

### DISCUSSION THREAD

**Discussion by dpolamarasetty** [osm-search/Nominatim#2910] [General]
Hi, I install the nomination server for all of us using
docker run -it \
  -e PBF_URL=https://download.geofabrik.de/north-america/us-latest.osm.pbf \
  -e REPLICATION_URL=https://download.geofabrik.de/north-america/us-updates/ \
   -e IMPORT_WIKIPEDIA=false \
   -e IMPORT_STYLE=full \
  -p 8080:8080 \
  --name nominatim \
  mediagis/nominatim:4.1

Now when i try to search for address
https://nominatim.openstreetmap.org/search.php?street=940%20cherry%20glen&city=fremont&state=ca&format=jsonv2
I get  2 records
 [{"place_id":305125791,"licence":"Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright","osm_type":"way","osm_id":6351421,"boundingbox":["37.561162575148","37.561262575148","-121.96946974406","-121.96936974406"],"lat":"37.561212575147515","lon":"-121.96941974406262","display_name":"940, Cherry Glen Circle, Fremont, Alameda County, California, 94536, United States","place_rank":30,"category":"place","type":"house","importance":-0.6399999999999999},{"place_id":305146265,"licence":"Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright","osm_type":"way","osm_id":6361520,"boundingbox":["37.562067761905","37.562167761905","-121.97021166667","-121.97011166667"],"lat":"37.562117761904766","lon":"-121.97016166666667","display_name":"940, Cherry Glen Terrace, Fremont, Alameda County, California, 94536, United States","place_rank":30,"category":"place","type":"house","importance":-0.6399999999999999}]

But when i do it on my local server, I get 3 records and don't get the house number in the display_name attribute,
http://XXX:8080/search.php?street=940%20cherry%20glen&city=fremont&state=ca&format=jsonv2
[
    {
        "place_id": 13785188,
        "licence": "Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright",
        "osm_type": "way",
        "osm_id": 6338607,
        "boundingbox": [
            "37.5615284",
            "37.5624282",
            "-121.9705925",
            "-121.9686257"
        ],
        "lat": "37.5621622",
        "lon": "-121.9697491",
        "display_name": "Cherry Glen Common, Fremont, Alameda County, California, 94536, United States",
        "place_rank": 26,
        "category": "highway",
        "type": "residential",
        "importance": 0.50001
    },
    {
        "place_id": 13900180,
        "licence": "Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright",
        "osm_type": "way",
        "osm_id": 6361520,
        "boundingbox": [
            "37.562015",
            "37.5623998",
            "-121.9703446",
            "-121.9699225"
        ],
        "lat": "37.5623998",
        "lon": "-121.9703446",
        "display_name": "Cherry Glen Terrace, Fremont, Alameda County, California, 94536, United States",
        "place_rank": 26,
        "category": "highway",
        "type": "residential",
        "importance": 0.50001
    },
    {
        "place_id": 13900674,
        "licence": "Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright",
        "osm_type": "way",
        "osm_id": 6351421,
        "boundingbox": [
            "37.5611565",
            "37.5618337",
            "-121.9696131",
            "-121.968514"
        ],
        "lat": "37.5611565",
        "lon": "-121.9692497",
        "display_name": "Cherry Glen Circle, Fremont, Alameda County, California, 94536, United States",
        "place_rank": 26,
        "category": "highway",
        "type": "residential",
        "importance": 0.50001
    }
]

I did a full import on the US data set. Can someone please let me know which data set i need to show the whole address with the house number,Unit etc

**Comment by mtmail** [comment]
https://nominatim.org/release-docs/latest/customize/Tiger/

You need to consult the nominatim-docker documentation (separate project) how that's handled in their setup or environment variables, I think I've seen it's possible there. Note the latest Nominatim release is 4.2.0

**Comment by dpolamarasetty** [comment]
Thank you mtmail. I will download the tiger data as well and try again.

**Comment by ernestomi** [comment]
Hi both!

I was having exactly the same issue, and can confirm that using `IMPORT_TIGER_ADDRESSES=true` adds the house number details that I needed. I hope the same works for you!

Also, maybe its worth posting the link to the repo that we are using since its a bit misleading that this discussion is on the Nominatim repo

https://github.com/mediagis/nominatim-docker

**Comment by dpolamarasetty** [comment]
hey @ernestomi ,yes I was able to resolve the issue after  adding -e IMPORT_TIGER_ADDRESSES=true  in the docker run cmd.
